Automattic/wp-calypso
# NUX: disable onboarding signup variation for the desktop app

This PR disables the new signup flow in the desktop app for now until we can perform further testing.

## Testing

1. Track AB test activation by pasting the following in your console: `localStorage.setItem( 'debug', 'calypso:abtests' );`
2. Enable the onboarding flow: `localStorage.setItem( 'ABTests', '{"improvedOnboarding_20181023" : "onboarding"}' );`
3. Visit `/start` and check to see that the AB test `improvedOnboarding_20181023` has an existing variation of "onboarding"
4. Visit `/start/about?flags=-signup/onboarding-flow` to disable the feature flag. Check to see that the AB test `improvedOnboarding_20181023` has an existing variation of "main"
